core/controllers/creator_dashboard.py

```diff
@@ -147,7 +147,8 @@ def get(self):
                 interaction_templates),
             'dependencies_html': jinja2.utils.Markup(dependencies_html)
         })
-        self.render_template('pages/creator_dashboard/creator_dashboard.html')
+        self.render_template('pages/creator_dashboard' +
```

Comment:
Don't add `+` to concatenate instead split after the opening brackets:

```
        self.render_template(
            'pages/creator_dashboard/creator-dashboard.mainpage.html')
```

Reply:
Hi @ankita240796 , I will make all changes after I get done with #6520 , as it will cause merge conflicts later and Also im stuck with another project atm. 
Thank you so much! 
-Amey

Reply:
Thanks for the update @amey-kudari!